webpack.config.js

```diff
@@ -236,7 +236,7 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					loader: 'html-loader',
 				},
 				{
-					test: /\.(svg)$/,
+					test: /\.(?:jpg|jpeg|png|svg)$/i,
```

Comment:
Why no `.gif`?

Reply:
De we need it? Happy to add it if so 👍

Reply:
I don't need a .gif at this moment, but I can imagine plenty of uses for animated gifs in blocks. Just seems like something that will mess someone up, someday, so unless there's a downside might as well support it out of the gate?